packages/cli/src/commands/init.ts

```diff
@@ -22,10 +22,11 @@ import { getTelemetry } from '../lib/telemetry.js';
 // Get the default prompt
 const { prompt } = enquirer;
 
-// Ensure .rover/ is in .gitignore
+// Ensure .rover/tasks/ and .rover/settings.json are in .gitignore
 const ensureGitignore = async (projectPath: string): Promise<void> => {
   const gitignorePath = join(projectPath, '.gitignore');
-  const roverEntry = '.rover/';
+  const roverTasksEntry = '.rover/tasks/';
+  const roverSettingsEntry = '.rover/settings.json';
```

Comment:
I would consider following the settings format of other agents. They keep the `settings.json` file as the project configuration and ignore the `settings.local.json` as the user preferences. Even we still use `rover.json` as the project settings, I will migrate into that direction.


```suggestion
  const roverSettingsEntry = '.rover/settings.local.json';
```